docker-compose.yml

```
@@ -38,6 +38,20 @@ services:
       timeout: 5s
       retries: 5
       start_period: 30s
-
+  nginx:
+    build: ./nginx
+    container_name: nginx
+    ports:
+      - "80:80"
+    restart: always
+    depends_on: 
```

yamllint

[error] trailing spaces

(trailing-spaces)